src/main/kotlin/no/nav/familie/ef/iverksett/infrastruktur/task/TaskFlyt.kt

```diff
@@ -45,8 +45,12 @@ fun Task.opprettNesteTask(): Task {
     return lagTask(nesteTask)
 }
 
-fun Task.opprettNestePubliseringTask(): Task {
-    val nesteTask = TaskType(this.type).nestePubliseringsflytTask()
+fun Task.opprettNestePubliseringTask(erMigrering: Boolean = false): Task {
+    val nesteTask = if (erMigrering) {
```

Comment:
Nesten så denne burde ha en sjekk for att det en en task som er `SendPerioderTilInfotrygdTask`, sånn att ikke fler tar den i bruk? 

Det finnes en kommentar på taskene litt lengre opp her `// går ikke videre ved migrering` som kanskje burde justeres :)

Reply:
Enig! Fikset